kahuna/public/js/image/controller.js

```diff
@@ -1,13 +1,12 @@
 import angular from 'angular';
 
 import '../image/service';
-import '../edits/service';
-import '../analytics/track';
 import '../components/gr-delete-image/gr-delete-image';
 import '../components/gr-add-label/gr-add-label';
 import '../downloader/downloader';
 import '../components/gr-crop-image/gr-crop-image';
 import '../components/gr-delete-crops/gr-delete-crops';
+import '../components/gr-image-metadata/gr-image-metadata';
```

Comment:
What are you thoughts around having these `components/componentname/componentname`?
Why not just `components/componentname`?